test/e2e/settings/pod_security_policy.go

```diff
@@ -38,7 +39,7 @@ const (
 	ingressControllerPSP = "ingress-controller-psp"
 )
 
-var _ = framework.IngressNginxDescribe("[Serial] Pod Security Policies", func() {
```

Comment:
Please don't remove [Serial]
This is used to run this particular test in isolation to avoid issues with PSP